distribution/server-quarkus/src/main/version.txt

```diff
@@ -0,0 +1 @@
+${product.name.full} - Version ${product.version}
```

Comment:
Change to "Keycloak.X - Version ${product.version}"

Reply:
Changed as requested